Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
Traceback (most recent call last):
  File "/media/sf_John_shared/loghawk/loghawk.py", line 4, in <module>
    with open("access.log") as f:
         ^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'access.log'
